v2/concordium/types.proto

```diff
@@ -607,6 +657,1084 @@ message Duration {
   uint64 value = 1;
 }
 
+// A reason for why a transaction was rejected. Rejected means included in a
+// block, but the desired action was not achieved. The only effect of a
+// rejected transaction is payment.
+message RejectReason {
+
+  message ModuleHashAlreadyExists {
+    ModuleRef contents = 1;
+  }
+
+  message InvalidAccountReference {
+    AccountAddress contents = 1;
```

Comment:
```suggestion
    AccountAddress address = 1;
```

Reply:
I actually just removed several of these simple wrapper types as I deemed them unnecessary.